data/in/config/DEA_queries/dea_queries.json

```diff
@@ -111,12 +119,20 @@
             "<meta>@sample?_study=[TCGA,GTEX]&_primary_site=[Thyroid Gland,Thyroid]&_sample_type=[Normal Tissue,Solid Tissue Normal]"
         ]
     },
-    "Lymphoma": {
+    "blood_cancer": {
+        "case": [
+            "<meta>@sample?_study=TCGA&_primary_site=[Blood,White blood cell]&_sample_type!=[Solid Tissue Normal,Control Analyte]"
+        ],
+        "control": [
+            "<meta>@sample?_study=[TCGA,GTEX]&_primary_site=[Blood,White blood cell]&_sample_type=[Normal Tissue,Solid Tissue Normal]"
+        ]
+    },
+    "bone_marrow_cancer": {
```

Comment:
For consistency with previous cancer types:
`bone_marrow_cancer` -> `Bone_Marrow_cancer`